qiskit_machine_learning/kernels/trainable_fidelity_quantum_kernel.py

```diff
@@ -0,0 +1,133 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Trainable Quantum Kernel"""
+
+from __future__ import annotations
+
+import numpy as np
+from qiskit import QuantumCircuit
+from qiskit.algorithms.state_fidelities import BaseStateFidelity
+from qiskit.circuit import Parameter, ParameterVector
+
+from .fidelity_quantum_kernel import FidelityQuantumKernel, KernelIndices
+from .trainable_kernel import TrainableKernel
+from ..exceptions import QiskitMachineLearningError
+
+
+class TrainableFidelityQuantumKernel(TrainableKernel, FidelityQuantumKernel):
+    r"""
+    Finding good quantum kernels for a specific machine learning task is a big challenge in quantum
+    machine learning. One way to choose the kernel is to add trainable parameters to the feature
+    map, which can be used to fine-tune the kernel.
+
+    This kernel has trainable parameters :math:`\theta` that can be bound using training algorithms.
+    The kernel entries are given as
+
+    .. math::
+
+        K_{\theta}(x,y) = |\langle \phi_{\theta}(x) | \phi_{\theta}(y) \rangle|^2
+    """
+
+    def __init__(
+        self,
+        *,
+        feature_map: QuantumCircuit | None = None,
+        fidelity: BaseStateFidelity | None = None,
+        training_parameters: ParameterVector | list[Parameter] | None = None,
```

Comment:
Generally we have been trying to be more generic in these hints so as not to tie things down to a list specifically but rather use Sequence so `Sequence[Parameter]`

Reply:
Thanks for spotting, I'm trying to replace with `Sequence` wherever possible.